### PR TITLE
Include ruby 2.6 in Travis test matrix

### PR DESCRIPTION
**Pull request by eitoball**
This PR includes ruby 2.6 in Travis CI test matrix.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,19 +14,10 @@ matrix:
   allow_failures:
     - rvm: ruby-head
     - rvm: jruby-head
+  exclude:
+    - rvm: 2.4.5
+    - env: BRANCH=master
   include:
-    - rvm: 2.2.10
```
Comment (rafaelfranca) [inline .travis.yml]: Any reason to remove those?

Comment (eitoball) [reply .travis.yml]: Because 2.2.10 and 2.3.8 are not in `rvm:`.

Comment (rafaelfranca) [reply .travis.yml]: Right, they are not in the main matrix and this is why we are adding in the `include:` matrix.

Comment (eitoball) [reply .travis.yml]: Ah... `include`... I was thinking of `exclude`

Comment (eitoball) [reply .travis.yml]: Fixed. I am not sure why build on Travis CI is failing. It is passing on https://travis-ci.org/eitoball/activeresource/builds/508732442

**Comment by amatsuda** [comment]
@eitoball I'm sorry I just pushed 9aa30b17d1201425e6bef2db4d829e01b8a6202b without noticing that this PR already exists. Could you rebase and push-f if you're still willing push the rest of this PR?

**Comment by eitoball** [comment]
@amatsuda I just rebased and force-pushed. Thank you for checking out.

**Comment by amatsuda** [comment]
@eitoball Thank you :)

**Pull request merged**